Project-Aurora/Project-Aurora/Devices/Uniwill/KeyboardDevice.cs

```diff
@@ -0,0 +1,346 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Aurora.Devices.RGBNet;
+using Aurora.Settings;
+using Aurora.Utils;
+using IronPython.Runtime.Operations;
+using keyboard;
+using Microsoft.Win32;
+using RegistryUtils;
```

Comment:
why do you need a different library for this? you could use `Microsoft.Win32.Registry`

Reply:
Thanks for your suggestion. I will change it.